__tests__/app.test.js

```diff
@@ -69,3 +70,23 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id", () => {
+  it("should respond with status 200 and return an article object, with article_id, author, title,  body, topic, created_at and votes", () => {
+    const article_id = 1;
+    return request(app)
+      .get(`/api/articles/${article_id}`)
+      .expect(200)
+      .then(({ body: { article } }) => {
+        expect(article).toEqual({
```

Comment:
this is a brittle test, if you later patched this article it wouldn't pass. Try using the jest matchers .toMatchObject or .objectContaining so that it doesn't break with later functionality